tekore/_convert.py

```diff
@@ -141,6 +141,8 @@ def from_url(url: str) -> Tuple[str, str]:
     """
     Parse type and ID from an URL.
 
+    Any GET parameters in the URL will be ignored.
```

Comment:
Good placement to have the separate section after the initial summary. With the change below I think this could also simply reference the tracking information.